src/Arcus.Messaging.Pumps.ServiceBus/Configuration/AzureServiceBusMessagePumpSettings.cs

```diff
@@ -209,142 +290,119 @@ public AzureServiceBusMessagePumpSettings(
         /// <summary>
         /// Gets the fully qualified namespace where the Azure Service Bus entity is located.
         /// </summary>
+        [Obsolete("Will be made 'private' in v3.0")]
```

Comment:
Is nowwhere being used. Actually, the entire 'settings' model should be made `internal`, I believe.